src/Tribe/Integrations/WPML/WPML.php

```diff
@@ -80,6 +84,40 @@ protected function hook_filters() {
 			$category_translation = Tribe__Events__Integrations__WPML__Category_Translation::instance();
 			add_filter( 'tribe_events_category_slug', array( $category_translation, 'filter_tribe_events_category_slug' ), 20, 2 );
 		}
+
+		//// WORK IN PROGRESS!!!
+
+		add_action( "tribe_repository_events_pre_get_posts", function () {
+			$request_uri = isset( $_SERVER['REQUEST_URI'] )
+				? filter_var( $_SERVER['REQUEST_URI'], FILTER_SANITIZE_STRING )
+				: false;
+			if(false === $request_uri){
+				return;
+			}
+			$url_language_code = false;
+			global $sitepress;
+			$languages               = array_column( (array) $sitepress->get_active_languages(), 'default_locale' );
+			$flags = I18n::COMPILE_STRTOLOWER;
+			$translated_bases = tribe( 'tec.i18n' )
+				->get_i18n_strings( ['archive'=> Tribe__Events__Main::instance()->rewriteSlug], $languages, ['default','the-events-calendar'], 'en_US', $flags );
```

Comment:
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Expected 1 space between "'archive'" and double arrow; 0 found
* :no_entry_sign: Expected 1 space before "=>"; 0 found
* :no_entry_sign: Expected 1 space before "=>"; 0 found
* :no_entry_sign: Missing space before array closer.
* :no_entry_sign: Missing space before array closer.
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Expected 1 space between comma and "'the-events-calendar'"; 0 found
* :no_entry_sign: Expected 1 space between comma and "'the-events-calendar'"; 0 found
* :no_entry_sign: Missing space before array closer.
* :no_entry_sign: Missing space before array closer.

<sub>hash: cc2d4b179996684b20435c56bb2edaac</sub>